package.json

```
@@ -406,6 +406,7 @@
           "@babel/preset-typescript",
           "@babel/generator",
           "@babel/traverse",
+          "@swc/helpers",
           "@rnx-kit/eslint-plugin",
```

_nit:_ Keep alphabetical order

```suggestion
          "@rnx-kit/eslint-plugin",
          "@swc/helpers",
```

---

[3abe9f4](https://github.com/microsoft/fluentui/pull/27250/commits/3abe9f4965551e533e073034ae745e667512ec1e)